app/src/main/java/edu/uc/groupproject/covid19tracker/service/CasesService.kt

```diff
@@ -9,12 +9,12 @@ import retrofit2.Response
 
 class CasesService {
     fun fetchGlobalData() : MutableLiveData<ArrayList<GlobalData>> {
-        var _globalData = MutableLiveData<ArrayList<GlobalData>>()
+        val _globalData = MutableLiveData<ArrayList<GlobalData>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(ICasesDao::class.java)
         val call = service?.getGlobalData()
         call?.enqueue(object: retrofit2.Callback<ArrayList<GlobalData>> {
-            override fun onFailure(call: Call<ArrayList<GlobalData>>, t: Throwable) {
-                println("Error recieved on API call: " + t.toString())
+            override fun onFailure(call: Call<ArrayList<GlobalData>>, error: Throwable) {
+                println("Error received on API call: $error")
```

Comment:
Proper concatenation: https://kotlinlang.org/docs/reference/basic-types.html

Reply:
Definitely... easier to do in Kotlin than in Java!